hack/build-go.sh

```diff
@@ -57,18 +61,18 @@ if [ $# -eq 0 ]; then
     if [ "${target}" = "test" ]; then
         (
             # Ignoring container-disk-v2alpha since it is written in C, not in go
-            go ${target} -v --ignore=container-disk-v2alpha ./cmd/...
+            go ${target} -v -tags "${KUBEVIRT_GO_BUILD_TAGS}" --ignore=container-disk-v2alpha ./cmd/...
         )
         (
-            go ${target} -v -race ./pkg/...
+            go ${target} -v -tags "${KUBEVIRT_GO_BUILD_TAGS}" -race ./pkg/...
         )
     else
         (
-            go $target -tags selinux ./pkg/...
-            GO111MODULE=off go $target ./staging/src/kubevirt.io/...
+            go $target -tags "selinux,${KUBEVIRT_GO_BUILD_TAGS}" ./pkg/...
```

Comment:
agree with @rmohr since you are introducing it, we can avoid the hardcoded tags and provides defaults

Reply:
`selinux` is used only in some places. When making it the default of `KUBEVIRT_GO_BUILD_TAGS` it would be used in places where it does not apply / is not wanted?

Reply:
hm I think, we apply it  to the whole codebase with bazel. It should have no effect on unrelated code. I would prefer to start simple with a more understandable base-behaviour if possible: "Set build tags with this env variable".

Reply:
Changed it :+1: